### DISCUSSION TITLE
New Issue Templates & Tasks for Curation Team

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/RiverscapesXML#129] [RAVE: Symbology/BusinessLogic] [answered]
Hello @Riverscapes/riverscape-project-curators team. To make it easier for you to remember all the steps necessary to follow through on for "curating" a layer, I have created a new issue template with all the tasks. This makes it easier for a) the likes of @wally-mac, @philipbaileynar, @nick4rivers and myself to make the request (and you too if you self-assign), b) you to consistently complete and leave the bread-crumb trail of the tasks without missing critical steps, c) each other to review, trouble shoot and/or figure out status; and d) split the standard tasks into other issues assigned to different members of the team.  The [video](https://www.youtu.be/watch?v=YLlN2z2o2ds) below spells out how this works and I'd encourage you to watch it or use as a reference:

[![Alt-text](https://img.youtube.com/vi/YLlN2z2o2ds/0.jpg)](https://www.youtube.com/watch?v=YLlN2z2o2ds)

Shout with questions.

**Comment by lauren-herbine** [comment, marked as answer]
Thanks for pulling these templates together @joewheaton! This will definitely help me/us track where we are on each layer and project type. Quick Q: when do you want us assigning individual tasks within the bulleted list just by tagging a person...

- [ ] like this - @joewheaton to-do

and when do you want us to be converting these into their own issues (like you did in the video to assign those three tasks to Shelby)?

**Reply by joewheaton** [reply]
Good question @lauren-herbine. No hard and fast rules on when to either.  Some folks like Matt and Philip don't like too many issues, but some issues have no accountability as multiple people are responsible and no one does it. If all tasks are done by one person in an issue there is obviously no need. If folks that are mentioned are doing what they need to and checking off task boxes, also no need. If issue is getting bloated and/or stuff not getting done as it is not assigned as its own issue to someone, then that's an opportune time to split. Make sense?

**Reply by lauren-herbine** [reply]
Makes sense, thanks!

**Reply by joewheaton** [reply]
Hey @Riverscapes/riverscape-project-curators. Thanks to @lauren-herbine and @shelbysawyer for progress since the workshop. Its been crickets from the rest of you. So @wally-mac and I will be laying out explicit assignments.

**Reply by wally-mac** [reply]
@Cashe93 and @Tanmanfalcon have been cranking on curation assignments this week after being out in the field last week

**Comment by Cashe93** [comment]
Apologies for my silence on this. I've spoke with Wally and look forward to getting some tasks lined out and completed.


________________________________
